java-frontend/src/test/files/se/ConditionAlwaysTrueOrFalseCheck.java

```diff
@@ -1325,6 +1325,18 @@ private Object nullableMethod() {
     return new Class1().method();
   }
 
+  private void annotationWithNullableAndNonnullReturn() {
+    Object foo = annotationWithNullableAndNonnull();
+    if(foo == null) { // compliant, nullable wins
+    }
+  }
+
+  @javax.annotation.Nullable
+  @javax.annotation.Nonnull
+  private Object annotationWithNullableAndNonnull() {
```

Comment:
With the other testcase is that one still relevant ?

Reply:
Yes. It shows explicitly our decision when we face a Nullable/Nonnull conflict.